egs/aishell/s10/local/run_chain.sh

```diff
@@ -221,110 +232,92 @@ if [[ $stage -le 13 ]]; then
         --output-dim $output_dim \
         --prefinal-bottleneck-dim $prefinal_bottleneck_dim \
         --subsampling-factor-list "$subsampling_factor_list" \
-        --train.cegs-dir exp/chain/merged_egs \
-        --train.ddp.world-size $num_gpus \
-        --train.den-fst exp/chain/den.fst \
+        --train.use-ddp $use_ddp \
+        --train.ddp.init-method $init_method \
```

Comment:
please sort the options alphabetically.

If you are using vim, you can use the method mentioned above.